### PR TITLE
Reconnect to RDS if needed

### PR DESCRIPTION
**Pull request by takai**
When RDS is restarted, fluent-plugin-rds-slowlog continues to fail with "closed MySQL connection" error until restarting td-agent.

This patch allow td-agent to try to reconnect when server seems to be down.

### CODE REVIEW

File: lib/fluent/plugin/in_rds_slowlog.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'fluent/input'
```
Comment (takai) [inline lib/fluent/plugin/in_rds_slowlog.rb]: fluentd v0.14.6 may require this line.

**Comment by winebarrel** [comment]
@takai Please rebase master (fix for travis ci test)

**Comment by takai** [comment]
@winebarrel rebased.

**Comment by winebarrel** [comment]
LGTM

**Comment by winebarrel** [comment]
Marged.
@kenjiskywalker Please comment if there is a problem 🙇

**Pull request merged**